runtime/development/src/lib.rs

```diff
@@ -2073,15 +2124,16 @@ construct_runtime!(
 		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 120,
 		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Config<T>, Event<T>, Origin} = 121,
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 122,
-		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 123,
+		// Removed: DmpQueue = 123
 		XTokens: pallet_restricted_xtokens::{Pallet, Call} = 124,
 		XcmTransactor: pallet_xcm_transactor::{Pallet, Call, Storage, Event<T>} = 125,
 		OrmlXTokens: orml_xtokens::{Pallet, Event<T>} = 126,
+		MessageQueue: pallet_message_queue::{Pallet, Call, Storage, Event<T>} = 127,
```

Comment:
Will this be the queue for ALL xcm messages or just DMP?

Reply:
All messages, XCMP, DMP, UMP: 

> This is intended to be used with an XCM adapter struct implementing the ProcessMessage trait, to be made at a later time in the Polkadot codebase as a single, general queue for XCMP, DMP, UMP and local dispatch/queuing.

https://github.com/paritytech/substrate/pull/12485